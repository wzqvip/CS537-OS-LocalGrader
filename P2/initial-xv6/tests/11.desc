testing getfilename with multiple files