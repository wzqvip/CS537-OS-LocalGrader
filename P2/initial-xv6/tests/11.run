../tester/run-xv6-command.exp CPUS=1 Makefile.test test_11 | grep XV6_TEST_OUTPUT
