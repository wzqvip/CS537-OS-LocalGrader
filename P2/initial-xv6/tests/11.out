XV6_TEST_OUTPUT Open filename: helloworld.txt
